### ISSUE TITLE
Automatic splitting status output

### ISSUE THREAD

**Issue by matz-e** [dmwm/CRABServer#5624]
The status for the automatic splitting currently displays only the status of the probe DAG. I've done some work that will display the status as follows:
```
Status on the scheduler:        Splitting probes:       COMPLETED
                                Processing jobs:        FAILED
                                Tail jobs:              COMPLETED (2/3), FAILED (1/3)

Probe jobs status:              finished      100.0% (5/5)

Jobs status:                    failed          3.9% ( 17/439)
                                finished       96.1% (422/439)

Tail jobs status:               failed          2.8% ( 1/36)
                                finished       97.2% (35/36)
```

This output is still misleading. [Following Jose's suggestion](https://hypernews.cern.ch/HyperNews/CMS/get/crabDevelopment/2698/1/1/2/1/1/1/2/1/1/1/1/1.html), I think this would be a good target format:
```
Status on the scheduler:        FAILED

Jobs status:                    failed          9.1% (  1/458)
                                finished       99.8% (457/458)
```
For the user, the following information is present:

* Task status is `FAILED` since at least one job failed
* 457 jobs are done (457 output files should be on the SE: 422 "processing" jobs, and 35 tail jobs)
* 1 job failed and can be resubmitted ("processing" jobs cannot be resubmitted)

I would suggest:

1. Have the `--long` status include the first, longer status message, with additional change:

   * specify in the job list that processing jobs are `failed (reprocessed by tails)` or similar. Maybe `terminated` is a better word?

2. Have regular status follow Jose's suggestion and then maybe combine server + scheduler status down the road?

I have a small patch to the TW that will make sure that all the DAG information is available to the status command, I will file a PR later, once I'm reasonably sure that the data required is not going to change.

**Comment by matz-e** [comment]
Current state, ready for PR, follows. I've removed the split for the DAG status, I don't think it adds much information, and has a lot more potential for confusion. Probe jobs will be marked `no output` and included, to be consistent with the dashboard. Failed processing jobs will be labeled as `recovered`, as the tail jobs do that, more or less.

Long status:
```
❯ TauRoast master matze@lxplus108 [7s] crab st --long crab_0067_automatic_test_wrapper2
CRAB project directory:		/afs/cern.ch/work/m/matze/ttH/CMSSW_8_0_26_patch1/src/ttH/TauRoast/crab_0067_automatic_test_wrapper2
Task name:			180119_135345:matze_crab_0067_automatic_test_wrapper2
Grid scheduler:			crab3-5@vocms059.cern.ch
Status on the CRAB server:	SUBMITTED
Task URL to use for HELP:	https://cmsweb.cern.ch/crabserver/ui/task/180119_135345%3Amatze_crab_0067_automatic_test_wrapper2
Dashboard monitoring URL:	http://dashb-cms-job.cern.ch/dashboard/templates/task-analysis/#user=matze&refresh=0&table=Jobs&p=1&records=25&activemenu=2&status=&site=&tid=180119_135345%3Amatze_crab_0067_automatic_test_wrapper2
Status on the scheduler:	COMPLETED

Probe jobs status:              no output     100.0% (5/5)

Jobs status:                    finished       99.6% (270/271)
				recovered       0.4% (  1/271)

Tail jobs status:               finished      100.0% (37/37)

No publication information (publication has been disabled in the CRAB configuration file)

Error Summary: (use crab status --verboseErrors for details about the errors)

  1 jobs failed with exit code 10040

Have a look at https://twiki.cern.ch/twiki/bin/viewauth/CMSPublic/JobExitCodes for a description of the exit codes.

Extended Job Status Table:

 Job State        Most Recent Site        Runtime   Mem (MB)      CPU %    Retries   Restarts      Waste       Exit Code
 0-1 no output    Unknown                 0:16:05        716         92          0          0    0:00:00               0
 0-2 no output    Unknown                 0:16:59        734         77          0          0    0:00:00               0
 0-3 no output    Unknown                 0:15:46        714         91          0          0    0:00:00               0
 0-4 no output    Unknown                 0:16:08        718         87          0          0    0:00:00               0
 0-5 no output    Unknown                 0:15:52        694         93          0          0    0:00:00               0
   1 finished     Unknown                 7:08:16        710         96          0          0    0:00:00               0
   2 finished     Unknown                 4:32:48        711         96          0          0    0:00:00               0
…
 217 finished     Unknown                 5:47:52        700         97          0          0    0:00:00               0
 218 recovered    Unknown                 0:01:02         16          3          0          0    0:00:00           10040
 219 finished     Unknown                 5:08:46        747         96          0          0    0:00:00               0
…
 1-1 finished     Unknown                 1:31:54        687         86          0          0    0:00:00               0
 1-2 finished     Unknown                 1:36:55        774         90          0          0    0:00:00               0
 1-3 finished     Unknown                 2:42:00        783         90          0          0    0:00:00               0
 1-4 finished     Unknown                 0:33:28        755         86          0          0    0:00:00               0
 3-1 finished     Unknown                 0:59:30        722         94          0          0    0:00:00               0
```

And short status:
```
❯ TauRoast master matze@lxplus108 crab st crab_0067_automatic_test_wrapper2
CRAB project directory:		/afs/cern.ch/work/m/matze/ttH/CMSSW_8_0_26_patch1/src/ttH/TauRoast/crab_0067_automatic_test_wrapper2
Task name:			180119_135345:matze_crab_0067_automatic_test_wrapper2
Grid scheduler:			crab3-5@vocms059.cern.ch
Status on the CRAB server:	SUBMITTED
Task URL to use for HELP:	https://cmsweb.cern.ch/crabserver/ui/task/180119_135345%3Amatze_crab_0067_automatic_test_wrapper2
Dashboard monitoring URL:	http://dashb-cms-job.cern.ch/dashboard/templates/task-analysis/#user=matze&refresh=0&table=Jobs&p=1&records=25&activemenu=2&status=&site=&tid=180119_135345%3Amatze_crab_0067_automatic_test_wrapper2
Status on the scheduler:	COMPLETED

Jobs status:                    finished      100.0% (307/307)

No publication information (publication has been disabled in the CRAB configuration file)
Log file is /afs/cern.ch/work/m/matze/ttH/CMSSW_8_0_26_patch1/src/ttH/TauRoast/crab_0067_automatic_test_wrapper2/crab.log
```

**Comment by matz-e** [comment]
@belforte, @hernand, comments on this? The changes I have ready include some resubmission things I still need to test.

**Comment by hernand** [comment]
Looks great. Very good idea about the short and long status display. 

Small suggestion: maybe "rescheduled" would be more appropriate than "recovered" (or even "rescheduled on tail", to be more clear, that maybe too long)

**Comment by matz-e** [comment]
@hernand `rescheduled` is a lot better, thanks! I was worried that `recovered` implied that the data was already processed again or something in that vain, so I wasn't very happy using it.

**Comment by matz-e** [comment]
Implemented in dmwm/CRABClient#4738. Remaining status merger migrated to dmwm/CRABClient#4734.
